YANG/ccamp/otn-tunnel/ietf-otn-tunnel.yang

```diff
@@ -661,10 +740,22 @@ module ietf-otn-tunnel {
     }
   }
 
+  /* Augment label-step of the label restrictions for the forwarding
+     direction of path-in-segment of secondary path */
+  augment "/te:te/te:tunnels/te:tunnel/"
+        + "te:secondary-paths/te:secondary-path/"
+        + "te:path-in-segment/te:label-restrictions/"
+        + "te:label-restriction/te:label-step/te:technology" {
+    description "OTN label.";
+    case otn {
+      uses l1-types:otn-label-step;
+    }
```

Comment:
why no augmentation for label-start and label-end?

Reply:
see comment above: clarified that the label-start and label-end augmentations are already defined. This PR is adding also the augmentations for the label-step.